lib/util.cc

```diff
@@ -73,19 +73,6 @@ xint64toa(int64_t num)
     return buf;
```

Comment:
Some `#include` statements in this used-to-be-C file can probably be upgraded (including the removal of their guards), but I do not insist on that.